tests/nix_tests/configs/default.jsonnet

```diff
@@ -109,6 +109,14 @@
             base_fee: '100000000000',
           },
         },
+        erc20: {
+          'token_pairs': [{
+            'erc20_address': '0x5db67696C3c088DfBf588d3dd849f44266ff0ffa',
+            'denom': 'aevmos',
+            'enabled': true,
+            'contract_owner': 1,
+          }],
+        }
```

Comment:
Do we need this in the default config?
Or is something specific for the str-v2 test cases?
'Cause if it is specific to str-v2, we can just add this on the str-v2.jsonnet instead

Reply:
@GAtom22 added this to the default since we will be adding a token pair for WEVMOS-EVMOS going forward.